crates/sui-framework/sources/governance/validator_set.move

```diff
@@ -350,19 +367,19 @@ module sui::validator_set {
         vector::length(&self.active_validators) - vector::length(&self.pending_removals) + table_vec::length(&self.pending_validators)
     }
 
-    /// Returns true iff `validator_address` is a member of the active validators.
-    public(friend) fun is_active_validator(
+    /// Returns true iff the address exists in active validators.
+    public(friend) fun is_active_validator_by_sui_address(
         self: &ValidatorSet,
         validator_address: address,
     ): bool {
         option::is_some(&find_validator(&self.active_validators, validator_address))
     }
 
-
     // ==== private helpers ====
 
     /// Checks whether `new_validator` is already in currently active validator list.
-    /// Two validators are identical if they share the same sui_address or same IP or same name.
+    /// It differs from `is_active_validator_by_sui_address` in that the former checks
+    /// only the sui address but this function looks at more metadata.
     fun is_currently_active_validator(self: &ValidatorSet, new_validator: &Validator): bool {
```

Comment:
I think it will be more clear if we name this something like`is_duplicate_with_active_validator`